hardware/Tesla.h

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include "DomoticzHardware.h"
+#include "TeslaApi.h"
+#include "../main/json_helper.h"
```

Comment:
Not sure if you need this include here... (json_helper.h)

Reply:
indeed, not needed. Remainder of a previous change. Will remove.